--- gringotts-1.2.10/src/Makefile.am.orig	2011-10-14 18:13:12.022903244 +0200
+++ gringotts-1.2.10/src/Makefile.am	2011-10-14 18:13:17.079567229 +0200
@@ -3,7 +3,6 @@ INCLUDES = \
 	-DG_LOG_DOMAIN=\"gringotts\" 			\
 	-DLOCALEDIR=\"$(datadir)/locale\" \
 	-DDOCDIR=\"$(docdir)\" \
-	-DG_DISABLE_DEPRECATED 					\
 	-DGDK_PIXBUF_DISABLE_DEPRECATED 		\
 	-DGDK_DISABLE_DEPRECATED 				\
 	-Wpointer-arith  \
