diff -urN aaa/gringotts-1.3.0pre5/gringotts.desktop.in gringotts-1.3.0pre5/gringotts.desktop.in
--- aaa/gringotts-1.3.0pre5/gringotts.desktop.in	Tue Jan 14 19:10:20 2003
+++ gringotts-1.3.0pre5/gringotts.desktop.in	Wed Apr 16 22:24:10 2003
@@ -10,12 +10,13 @@
 Type=Application
 TryExec=
 X-GNOME-DocPath=
-Name[it_IT]=@GRG_CAP_NAME@
-GenericName[it_IT]="Cassaforte" elettronica
-Comment[it_IT]=Una cassaforte elettronica
-Name[fr]=@GRG_CAP_NAME@
+GenericName[it]="Cassaforte" elettronica
+Comment[it]=Una cassaforte elettronica
 GenericName[fr]="Coffre-fort" électronique
 Comment[fr]=Un coffre-fort électronique
-Name[de]=@GRG_CAP_NAME@
 GenericName[de]=Elektronischer "Tresor"
 Comment[de]=Ein elektronischer "Tresor"
+GenericName[pl=Elektroniczny strażnik
+Comment[pl]=Elektroniczny strażnik
+Categories=GTK;Utility;
+# vi: encoding=utf-8
